makeflow/src/makeflow.c

```diff
@@ -1500,12 +1444,27 @@ int main(int argc, char *argv[])
 
 	/* Remote storage modes do not (yet) support measuring storage for garbage collection. */
 
-	if(batch_queue_type==BATCH_QUEUE_TYPE_CHIRP || batch_queue_type==BATCH_QUEUE_TYPE_HADOOP) {
+	if(!batch_queue_supports_feature(remote_queue, "gc_size")) {
```

Comment:
I rather have: if( makeflow_gc_method == MAKEFLOW_GC_SIZE && !batch_queue_supports_feature(remote_queue, "gc_size")) {
...

The meaning is clearer.